debian/control

```diff
@@ -0,0 +1,37 @@
+Source: jetson-gpio
+Section: python
+Priority: optional
+Maintainer: NVIDIA Corporation
```

Comment:
This needs to be as follows, at least for launchpad.net to accept the package into a PPA:
Maintainer: NVIDIA Corporation <linux-tegra-bugs@nvidia.com>
(i.e. add a valid email address)